aptos-move/aptos-vm/src/transaction_validation.rs

```diff
@@ -189,40 +210,65 @@ fn run_epilogue(
     fee_statement: FeeStatement,
     txn_data: &TransactionMetadata,
     features: &Features,
+    required_deposit: Option<u64>,
```

Comment:
nit: maybe we can merge this into TransactionMetadata so we don't need to pass the extra param all over the place

Reply:
we now have `required_deposit` in `TransactionMetadata`.